parlai/core/metrics.py

```diff
@@ -18,6 +18,11 @@
 
 import re
 
+DEFAULT_METRICS = set(['correct', 'bleu', 'accuracy', 'f1'])
```

Comment:
you can also do `DEFAULT_METRICS = {'correct', 'bleu', ...}`. The syntax is a bit prettier.

Reply:
that is cool! Thanks!